pom.xml

```diff
@@ -43,6 +43,13 @@
                     <compilerArgs>
                         <arg>-parameters</arg>
                     </compilerArgs>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
+                            <version>1.18.22</version>
```

Comment:
Is adding lombok necessary?  I personally don't like adding it to libraries

Reply:
Isn't Lombok already a part of your project? My IntelliJ IDE automatically suggested upgrading the version